packages/connect-core/src/entities/App.ts

```diff
@@ -120,4 +97,28 @@ export default class App extends CoreEntity implements AppData {
   async roles(): Promise<Role[]> {
     return this._connector.rolesForAddress(this.address)
   }
+
+  get artifact(): AragonArtifact {
+    return this.#metadata[0] as AragonArtifact
+  }
+
+  get manifest(): AragonManifest {
+    return this.#metadata[1] as AragonManifest
+  }
+
+  get abi(): Abi {
```

Comment:
I only include getter for really useful properties. Otherwise, the dev can get them from the metadata files.